src/status_im/contexts/wallet/send/events.cljs

```diff
@@ -50,9 +50,11 @@
 
 (rf/reg-event-fx :wallet/select-send-address
  (fn [{:keys [db]} [{:keys [address token stack-id]}]]
-   {:db (assoc-in db [:wallet :ui :send :to-address] address)
-    :fx [[:navigate-to-within-stack
-          (if token [:wallet-send-input-amount stack-id] [:wallet-select-asset stack-id])]]}))
+   (let [[prefix to-address] (utils/split-prefix-and-address address)]
+     {:db (-> db
+              (assoc-in [:wallet :ui :send :to-address] to-address)
+              (assoc-in [:wallet :ui :send :address-prefix] prefix))
+      :fx [[:navigate-to-within-stack (if token [:wallet-send-input-amount stack-id] [:wallet-select-asset stack-id])]]})))
```

Comment:
Better for readability to add some linebreaks to this `if` (as the usual style).